cloud-architecture/custom-apps-and-consolidation/mysql-mds/files/terraform.tfvars

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+# Purpose: terraform variable definition file.
+# @author: Vijay Kokatnur
+
+# Identity and access parameters
+home_region     = "eu-frankfurt-1"
+region          = "eu-frankfurt-1"
+
+
+# general oci parameters
+compartment_id = "ocid1.compartment.oc1.."
+label_prefix   = "none"
+
+# ssh keys
+ssh_private_key_path = "keys/id_rsa"
+# ssh_public_key       = ""
+# ssh_public_key_path  = "none"
+ssh_public_key_path = "keys/id_rsa.pub"
+
+#networking parameters
+subnet_id     = "ocid1.subnet.oc1..."
+
+create_subnet   = false
+# If create_subnet = true - provide vcn_id, ig_route_id, newbits, netnum for subnet cidr 
+# newbits = 8
+# netnum  = 2
+#vcn_id        = ""
+#ig_route_id   = ""
+
+assign_public_ip = false
+enable_ext      = false
+
+# Compute Instance
+instance_count = 3
+image_id    = "ocid1.image.oc1..."
+os_version  = "8"
+shape = {
+  shape            = "VM.Standard2.4",
+  ocpus            = 4,
+  memory           = 60,
+  boot_volume_size = 100
+}
+state    = "RUNNING"
+timezone = "Etc/UTC"
+type     = "public"
+#upgrade_bastion  = false
+
+#MySQL DB System Paramters
+mysql_shape_name  = "MySQL.VM.Standard2.4.60GB"
+
+mysql_db_subnet  = "ocid1.subnet.oc1..."
+
+mysql_db_system_admin_username  = "admin" 
+
+mysql_db_system_admin_password  = "admin123"
```

Comment:
NO PASSWORD IN CODE!!!

Reply:
Will this format work - for eg:   username=<USERNAME> , password=<PASSWORD>